.vscode/tasks.json

```diff
@@ -35,7 +35,7 @@
                 "watch",
                 "run",
                 "--project",
-                "${workspaceFolder}/src/Azure.Functions.Cli/Azure.Functions.Cli.csproj"
+                "${workspaceFolder}/src/Cli/func/Azure.Functions.Cli.csproj"
```

Comment:
Thoughts on renaming `Azure.Functions.Cli.csproj` to func?

Reply:
I kind of like the project name being `Azure.Functions.Cli`